src/QSymbolicsBase/basic_ops_homogeneous.jl

```diff
@@ -40,6 +40,11 @@ function Base.:(*)(c::U, x::Symbolic{T}) where {U<:Union{Number, Symbolic{<:Numb
         SScaled{T}(c, x)
     end
 end
+
+function Base.zero(x::Symbolic{T}) where T
```

Comment:
This is a type piracy because you are adding a method to a function we do not own (zero) for a type we do not own (Symbolic). You probably want to do this for `QObj`